communicator/ssh/provisioner_test.go

```diff
@@ -105,6 +110,7 @@ func TestProvisioner_connInfoHostname(t *testing.T) {
 				"user":        "root",
 				"password":    "supersecret",
 				"private_key": "someprivatekeycontents",
+				"certificate": "somecertificate",
```

Comment:
Let's remove `somecertificate` from one of these 3 tests, and use that test to verify that `conf.Certificate` remains an empty string. Because it's optional, I'd like to see at least one test verify the exclusion still works as expected.

Reply:
done!